pkg/reconciler/members.go

```diff
@@ -223,14 +223,16 @@ func handleCreatingMembers(
 		go func(m *kdv1.MemberStatus) {
 			defer wgSetup.Done()
 			configmeta := configmetaGenerator(m.Pod)
+			// Set member state to configError. On successful return from this function
+			// state will be set to configured.
+			m.State = string(memberConfigError)
```

Comment:
So with this change, we never keep anything in creating state across multiple handler passes. That means we *could* get rid of the definition of the memberConfigured state altogether and just set things directly to memberReady when they succeed.

Howwwwwwever. With PR #62 I'm adding another reason that we will keep members in creating state. So I think this is good. Leave it like this and I'll just need to figure out how best to merge so that we handle the three different kinds of states that the gofunc can leave the member in.